Core/include/Acts/Utilities/Range1D.hpp

```diff
@@ -0,0 +1,322 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2016-2018 CERN for the benefit of the Acts project
```

Comment:
2021.

Thanks for the renaming!

Reply:
Ah of course, silly me. Should that be `2016-2021`, `2021-2021`, or `2021`?